ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -85,6 +86,7 @@ public boolean accept(File dir, String name) {
   private final AccountService accountService;
   private volatile boolean isActive = false;
   private PersistentIndex srcIndex;
+  private Set<Pair<Short,Short>> selectedContainers;
```

Comment:
minor: make this `private final`, then remove line 136

Reply:
Updated to add final. May I know why I need to remove line 136? The selectedContainers still need to be initialized right?